portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACProperties.macro

```diff
@@ -34,6 +34,15 @@ definition {
 		}
 	}
 
+	@summary = "Asserts the list of properties shown in the Settings"
+	macro assertPropertySettingsList {
+		for (var key_propertyName : list "${propertyList}") {
+			AssertElementPresent(
+				itemName = "${key_propertyName}",
+				locator1 = "ACSettings#ITEM_ON_LIST");
+		}
+	}
```

Comment:
Hi @91nilo, I think you should merge `assertPropertySettingsList` to `assertDataSourceName` and create a macro called `assertItemName`. Because `assertPropertySettingsList` and `assertDataSourceName` have basically the same function. Remember to change the **test case** that uses `assertDataSourceName`

```suggestion
	macro assertItemName {
		for (var itemName : list "${itemNameList}") {
			AssertElementPresent(
				itemName = "${itemName}",
				locator1 = "ACSettings#ITEM_ON_LIST");
		}
	}
```

Reply:
Hey @ViniLopes20 i did the changes. Thanks!